src/views/detail/detail.pug

```diff
@@ -14,12 +14,15 @@ html(lang='en')
     #app.container(v-cloak)
       sidebar(:is-user-logged-in='isUserLoggedIn' @list='changeCurrentList')
       .details-container
-        .row.space-between.baseline
-          h1.component-title
-            | {{ componentTitle }}
-          .author
+        h1.title.m-bottom-5
+          | {{ componentTitle }}
         #host.component-render-container
-        .code-container.row
-          code-snippet.item-1(lang='html' title='HTML' :code='html')
-          code-snippet.item-1(lang='scss' title='SCSS' :code='scss')
+        .code-container
+          .row
+            button.button-type(type='button' @click='changeCurrentCode("html")' v-bind:class='{ active: currentCode == "html" }')
```

Comment:
In Vue, there's no need to write the whole `v-bind:`, you can just use the shorthand `:`. So in this case, it would be `:class='{ active: currentCode == "html" }'`